libcontainer/dmz/dmz_unsupported.go

```diff
@@ -1,5 +1,5 @@
-//go:build !linux || runc_nodmz
-// +build !linux runc_nodmz
+//go:build !linux || (!386 && !amd64 && !arm && !arm64 && !loong64 && !ppc64le && !riscv64 && !s390x) || runc_nodmz
+// +build !linux !386,!amd64,!arm,!arm64,!loong64,!ppc64le,!riscv64,!s390x runc_nodmz
```

Comment:
idem, I think we don't need to add specific arches here, just leave the build tag.